python/core/auto_generated/qgsauxiliarystorage.sip.in

```diff
@@ -288,14 +288,21 @@ Returns the path of the current database used. It may be different from
 the target filename if the auxiliary storage is opened in copy mode.
 %End
 
-    bool saveAs( const QString &filename ) const;
+    QString errorString() const;
+%Docstring
+Returns the underlying error string when something went wrong.
```

Comment:
Would it be worth replacing "something went wrong" with something more explicit (and referencing to the `saveAs` method)?

Reply:
I updated the documentation accordingly. Let me know if you want me to modify something else or if it's not clear enough.